tests/pytorc_train_tests.py

```diff
@@ -12,34 +11,58 @@
+        self.full_transformer_params = {"use_decoder": True, "model_params": {"number_time_series": 3, "seq_length": 20,
+                                                                              "output_seq_len": 15},
+                                        "dataset_params": {"forecast_history": 20, "class": "default",
+                                                           "forecast_length": 15,
+                                                           "relevant_cols": ["cfs", "temp", "precip"],
+                                                           "target_col": ["cfs"], "interpolate": False, "train_end": 50,
+                                                           "valid_end": 100},
+                                        "training_params": {"optimizer": "Adam", "lr": .01, "criterion": "MSE",
+                                                            "epochs": 1, "batch_size": 2, "optim_params": {}},
+                                        "inference_params": {"hours_to_forecast": 10},
+                                        "wandb": False}
+        self.simple_param = {"use_decoder": True,
+                             "model_params": {"n_time_series": 3, "seq_length": 80, "output_seq_len": 20},
+                             "dataset_params": {"forecast_history": 20, "class": "default", "forecast_length": 15,
+                                                "relevant_cols": ["cfs", "temp", "precip"], "target_col": ["cfs"],
+                                                "interpolate": False, "train_end": 50,
+                                                "valid_end": 100}, "inference_params": {"hours_to_forecast": 10},
+                             "training_params": {"optimizer": "Adam", "lr": .01, "criterion": "MSE", "epochs": 1,
+                                                 "batch_size": 2, "optim_params": {}},
+                             "wandb": False}
+        self.transformer = PyTorchForecast("SimpleTransformer", self.keag_file, self.keag_file, self.keag_file,
+                                           self.full_transformer_params)
+        self.simple_linear_model = PyTorchForecast("SimpleLinearModel", self.keag_file, self.keag_file, self.keag_file,
+                                                   self.simple_param)
         self.opt = torch.optim.Adam(self.dummy_model.model.parameters(), lr=0.0001)
         self.criterion = torch.nn.modules.loss.MSELoss()
         self.data_loader = DataLoader(self.dummy_model.training, batch_size=2, shuffle=False, sampler=None,
-           batch_sampler=None, num_workers=0, collate_fn=None,
-           pin_memory=False, drop_last=False, timeout=0,
-           worker_init_fn=None)
-        # TODO add LSTM test self.lstm = 
+                                      batch_sampler=None, num_workers=0, collate_fn=None,
+                                      pin_memory=False, drop_last=False, timeout=0,
+                                      worker_init_fn=None)
```

Comment:
Did anything change here?

Reply:
no, just white spaces (it's the indent thing in pep8)